### ISSUE TITLE
Add SDE SearchParameter to CR plugin

### ISSUE THREAD

**Issue by rob-reynolds** [DBCG/cqf-ruler#559]
SDE has been re-designed. The cr plugin needs to have a corresponding SearchParameter resource loaded.

IS: the cr plugin does not include a SearchParameter for the supplemental data extension.

SHOULD BE: if the cr plugin is enabled a SearchParameter resource should be loaded for the supplemental data extension.

The SearchParameter resource has been added as a comment.

**Comment by rob-reynolds** [comment]
```
{
    "resourceType": "SearchParameter",
    "id": "deqm-measurereport-supplemental-data",
    "url": "http://hl7.org/fhir/us/davinci-deqm/SearchParameter/measurereport-supplemental-data",
    "version": "0.1.0",
    "name": "DEQMMeasureReportSupplementalData",
    "status": "active",
    "date": "2022-07-16",
    "publisher": "HL7 International - Clinical Quality Information Work Group",
    "contact": [
        {
            "telecom": [
                {
                    "system": "url",
                    "value": "http://www.hl7.org/Special/committees/cqi/index.cfm"
                }
            ]
        }
    ],
    "description": "Returns resources (supplemental data) from references on extensions on the MeasureReport with urls matching http://hl7.org/fhir/us/davinci-deqm/StructureDefinition/extension-supplementalData.",
    "jurisdiction": [
        {
            "coding": [
                {
                    "system": "urn:iso:std:iso:3166",
                    "code": "US",
                    "display": "United States of America"
                }
            ]
        }
    ],
    "code": "supplemental-data",
    "base": [
        "MeasureReport"
    ],
    "type": "reference",
    "expression": "MeasureReport.extension('http://hl7.org/fhir/us/davinci-deqm/StructureDefinition/extension-supplementalData').value",
    "xpath": "f:MeasureReport/f:extension[@url='http://hl7.org/fhir/us/davinci-deqm/StructureDefinition/extension-supplementalData'].value",
    "xpathUsage": "normal"
}
```

**Comment by rob-reynolds** [comment]
Closed by #564.
